backend/btrixcloud/models.py

```diff
@@ -977,16 +988,113 @@ class OrgQuotas(BaseModel):
 
 
 # ============================================================================
-class OrgCreate(BaseModel):
-    """Create a new org"""
+class SubscriptionEventOut(BaseModel):
+    """Fields to add to output models for subscription events"""
 
-    name: Optional[str] = None
-    slug: Optional[str] = None
+    oid: UUID
+    timestamp: datetime
+
+
+# ============================================================================
+class SubscriptionCreate(BaseModel):
+    """create new subscription"""
+
+    type: Literal["create"] = "create"
 
-    firstAdminInviteEmail: Optional[str] = None
+    subId: str
+    status: str
+    planId: str
+
+    firstAdminInviteEmail: str
     quotas: Optional[OrgQuotas] = None
 
-    subData: Optional[Dict[str, Any]] = None
+
+# ============================================================================
+class SubscriptionCreateOut(SubscriptionCreate, SubscriptionEventOut):
+    """Output model for subscription creation event"""
+
+
+# ============================================================================
+class SubscriptionUpdate(BaseModel):
+    """update subscription data"""
+
+    type: Literal["update"] = "update"
+
+    subId: str
+    status: str
+    planId: str
+
+    futureCancelDate: Optional[datetime]
```

Comment:
Can this be any date time format, or UTC?

Reply:
It should be datetime UTC, believe it will return an ISO string in UTC.